src/prefect/filesystems.py

```diff
@@ -894,16 +894,18 @@ async def get_directory(
             local_path: A local path to clone to; defaults to present working directory.
         """
         # CONSTRUCT COMMAND
-        cmd = f"git clone {self._create_repo_url()}"
+        cmd = ["git", "clone", self._create_repo_url()]
         if self.reference:
-            cmd += f" -b {self.reference}"
+            cmd.append("-b")
+            cmd.append(self.reference)
 
         # Limit git history
-        cmd += " --depth 1"
+        cmd.append("--depth")
+        cmd.append("1")
```

Comment:
It _could_ be more readable to use concatenation here:

```suggestion
        cmd += ['--depth', '1']
```

Reply:
Another way is to use `shlex.split` when finished joining the string.

Reply:
Both of those are cleaner solutions - updated!